src/ICal/ICal.php

```diff
@@ -2402,7 +2411,7 @@ public function isValidCldrTimeZoneId($timeZone, $doConversion = false)
      * @param  string $date
      * @param  string $duration
      * @param  string $format
-     * @return integer|DateTime
+     * @return integer|/DateTime
```

Comment:
Shouldn't this be `\DateTime`?

Reply:
Yes, but interestingly IntelliJ 2018 linter accepts either slash in the function documentation. Fixed this and will commit a change at some point.